presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix ``IS DISTINCT FROM`` for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve performance of array_join.
+* Improve efficiency of ``ST_Buffer``. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances :func:`ST_Buffer`.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Improve efficiency by using JTS instead of Esri for many geometrical operations and fix incorrect calculation of extreme points in certain cases (:issue:`14031`).
+* Keep sum stat for string when min/max stat is too long in ORC writer.
+* Add support for get partition names by filter.
```

Comment:
This isn't a user focused description @ajaygeorge  @arhimondr is this specific to the Hive connector? Is there an SPI change that goes with this? 

Would something like
`Add method `getPartitionNamesByFilter` to HiveMetastore SPI to get partitions based on a range predicate.  

Also, if it's specific to the HiveMetastore, it belongs under "Hive Changes"

Reply:
@rschlussel I think you are right. 

- It should be under "Hive Changes" .
- There are no changes to `presto-spi` for this.
- Description can be : Add method `getPartitionNamesByFilter` to HiveMetastore to get partitions based on predicates.

@arhimondr , It would be good if you can take a quick look as well.

Reply:
This change is not user visible, and doesn't change anything in practice. I wonder if we should add this to the release notes all together.

Reply:
ok will remove from the release notes